packages/dialog/src/native/dialog-native.tsx

```
@@ -0,0 +1,76 @@
+function Description(props: DescriptionProps) {
+  if (process.env.NODE_ENV === 'development') {
+    console.log('`Description` from @rn-primitives/dialog/native is only supported on native.');
+  }
+  return null;
+}
+
+const useRootContext = () => {
+  throw new Error(
+    'Cannot access the native useRootContext on the web. Please import from `@rn-primitives/dialog` or `@rn-primitives/dialog/web`'
+  );
+  return {} as RootContextReturnType;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)